tests/test-buttercup.el

```diff
@@ -266,7 +269,7 @@
               "su1 su2"))))
 
 (describe "The `buttercup-spec-full-name' function"
-  (let (su1 su2 sp1)
+  (let (su1 su2 sp1 sp2)
```

Comment:
On a brighter note, this is what linters are there for: Catching bugs and improving readability! Nice. :-)